src/components/loop-form.tsx

```diff
@@ -1,44 +1,52 @@
-import { makeStyles, TextField, Button } from '@material-ui/core';
+import { makeStyles, TextField, Typography, Button } from '@material-ui/core';
 import Autocomplete from '@material-ui/lab/Autocomplete';
 import React, { useState } from 'react';
 import { successMsg } from '../util/alerts';
+import fakePlaces from '../mock/data/places';
 
-interface robotLoopFormProps {
-  fleetName: string;
+interface LoopFormProps {
+  fleets: string[];
   requestLoop(
     fleetName: string,
     numLoops: number,
     startLocationPoint: string,
     endLocationPoint: string,
   ): void;
-  listOfPlaces: string[];
 }
 
-export const RobotLoopForm = (props: robotLoopFormProps) => {
-  const { requestLoop, fleetName, listOfPlaces } = props;
+export const LoopForm = (props: LoopFormProps) => {
+  const { requestLoop, fleets } = props;
   const classes = useStyles();
 
+  const [targetFleetName, setTargetFleetName] = useState(fleets.length >= 1 ? fleets[0] : '');
   const [numLoops, setNumLoops] = useState(0);
   const [startLocation, setStartLocation] = useState(
-    listOfPlaces.length >= 2 ? listOfPlaces[0] : '',
+    fakePlaces()[targetFleetName].length >= 2 ? fakePlaces()[targetFleetName][0] : '',
   );
   const [finishLocation, setFinishLocation] = useState(
-    listOfPlaces.length >= 2 ? listOfPlaces[1] : '',
+    fakePlaces()[targetFleetName].length >= 2 ? fakePlaces()[targetFleetName][1] : '',
   );
```

Comment:
oh I believe this happens because Fleet1 and Fleet2 do not have any places listed under `places.ts` 
![image](https://user-images.githubusercontent.com/28668944/87617309-ce8b5800-c749-11ea-847f-c05eb5c4499e.png)
would adding fleet1 and 2 in be considered good practice or there is another way to do this?

Reply:
Yea, just add it for now, this whole `fakePlaces` is a hacky workaround to an ongoing problem here #54

Reply:
Yeah, It's ok, but It'll be good to add a safeguard to the form I the meanwhile. Also, you can add an error saying something like
 * There are no places available for this fleet. Pick another fleet. or
 * This fleet is not enabled to go anywhere. Pick another fleet

You can take this as an example https://github.com/osrf/romi-dashboard/blob/2d22429068d014bc7257b0a28cbe8d0fa2c93498/src/components/robot-item-delivery-form.tsx